scipy/signal/tests/test_windows.py

```diff
@@ -147,16 +154,20 @@ def test_exponential():
 
 tukey_data = {
     (4, 0.5, True): array([0.0, 1.0, 1.0, 0.0]),
-    (4, 0.9, True): array([0.0, 0.84312081893436686, 0.84312081893436686, 0.0]),
+    (4, 0.9, True): array([0.0, 0.84312081893436686,
+                           0.84312081893436686, 0.0]),
     (4, 1.0, True): array([0.0, 0.75, 0.75, 0.0]),
     (4, 0.5, False): array([0.0, 1.0, 1.0, 1.0]),
-    (4, 0.9, False): array([0.0, 0.58682408883346526, 1.0, 0.58682408883346526]),
+    (4, 0.9, False): array([0.0, 0.58682408883346526,
+                            1.0, 0.58682408883346526]),
     (4, 1.0, False): array([0.0, 0.5, 1.0, 0.5]),
     (5, 0.0, True): array([1.0, 1.0, 1.0, 1.0, 1.0]),
-    (5, 0.8, True): array([0.0, 0.69134171618254492, 1.0, 0.69134171618254492, 0.0]),
+    (5, 0.8, True): array([0.0, 0.69134171618254492,
+                           1.0, 0.69134171618254492, 0.0]),
```

Comment:
These all look like PEP8 changes. Did you add any new tests for the `sym` case changes?